docs/site/_april_milestone.md

```diff
@@ -0,0 +1,75 @@
+# April Milestone
+
+## Overview
+
+With COVID-19 impacting the maintainers (like everyone else in the world!) at
+different levels, we will be committing less story points to give us more
+flexibility.
+
+The goals for this month would be to:
+
+- Aim at finish the
+  [Migration guide epic](https://github.com/strongloop/loopback-next/issues/3935)
+  excpet for the
+  [migrate LB3 components related tasks](https://github.com/strongloop/loopback-next/issues/3955).
+- Finish
+  [From model definition to REST API with no custom repository/controller classes epic](https://github.com/strongloop/loopback-next/issues/3935)
+
+## Goals
+
+- [ ] Migration guide #3935
+
+  - [ ] Migration Guide: Boot up #4838
+  - [ ] Migration Guide: Custom validation #4837
+  - [ ] Spike: How to migrate LB3 components #4099
+  - [ ] How to map LB3 CLI commands to LB4 CLI #3953
+  - [ ] Spike: How to migrate apps using loopback-component-oauth2 #3959
+  - [ ] How to run LB3 app tests when the app is mounted in a LB4 project #3978
+
+- [ ] From model definition to REST API #3935
+
+  - [ ] Docs for exposing REST API of a Model with no custom classes #2740
+
+- [ ] APIC/LB integration
+
+  - [ ] [Spike] End to End test Shopping example in cloud with APIConnect #4498
+
+- [ ] Documentation
+
+  - [ ] Docs: Add "Working with Data" section #2527
+  - [ ] Docs: how to use OpenAPI connector #4488
+
+- PR reviews
+  - [ ] feat(repository): hasManyThrough #4438
+
```

Comment:
@deepakrkris, can we also add the following task as well pls? Thanks.
Extract the jwt authentication to an extension module #4903